14/umbraco-cms/extending/customize-backoffice/development-flow/vite-package-setup.md

```diff
@@ -164,32 +173,28 @@ Learn more about the abilities of the manifest file in the [Umbraco Package Mani
 
 To be able to test your package, you will need to run your site.
 
-Before you do this, you need to include all the files in the `src` folder and the `umbraco-package.json` file in your project.
+Before you do this, you need to make sure to run `npm run build` to compile your TypeScript files and copy them to the `App_Plugins/Client` folder.
 
-If you try to include these resources via Visual Studio (VS), then only the `dist` folder needs to be included. Otherwise, VS will try to include a few lines on your `.csproj` file to compile the TypeScript code that exists in your project folder. When you run your website, VS will try to compile these files and fail.
+If you try to include some of these resources via Visual Studio (VS), then make sure not to include TypeScript files. Otherwise, VS will try to include a few lines on your `.csproj` file to compile the TypeScript code that exists in your project folder. When you run your website, VS will try to compile these files and fail.
```

Comment:
Would it make sense to add this to a warning instead?
Otherwise I think it would be easy to miss.